packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -127,7 +127,8 @@ export class DataSyncPlugin extends GraphbackPlugin {
 
     modelTC.addFields({
       [DataSyncFieldNames.lastUpdatedAt]: {
```

Comment:
@wtrocki this field corresponds to https://github.com/aerogear/graphback/blob/e450454bac3d9d05cca0e5478e447c9e09395791/packages/graphback-datasync/src/util.ts#L39 which is `_lastUpdatedAt`. So it is not the generic one that's got indexed.

Reply:
We can also see this from the index https://github.com/aerogear/graphback/pull/1863/files#diff-a7d16cb5768bfd64feb81ba0b1a8921bR185